Core/src/main/java/com/plotsquared/core/generator/HybridUtils.java

```diff
@@ -175,8 +171,20 @@ public void analyzeRegion(final String world, final CuboidRegion region, final R
                 }
             }
 
-            System.gc();
-            System.gc();
+            BlockArrayCacheScopedQueueCoordinator oldBlockQueue = new BlockArrayCacheScopedQueueCoordinator(
+                    oldBlocks,
+                    area.getMinGenHeight(),
+                    Location.at("", region.getMinimumPoint()),
+                    Location.at("", region.getMaximumPoint())
+            );
+
+            region.getChunks().forEach(blockVector2 -> {
```

Comment:
the lambda parameter should probably be called something like `chunkPos` to avoid confusion